lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -157,51 +157,34 @@ def patch_action_controller
 
     def patch_process_action
       do_once(:patch_process_action) do
+        ActionControllerPatch.instrument(::ActionController::Metal)
+      end
+    end
+
+    # Compat module for Ruby versions not supporting #prepend
+    module CompatActionControllerPatch
```

Comment:
`Compat` should be expanded into `Compatible` or `Compatibility` for readability.

Reply:
👍